pages/components/navigation/navigation.css

```diff
@@ -38,3 +44,93 @@
 	cursor: pointer;
 	color: var(--colour-primary);
 }
+
+.navbar-cta > *:hover:last-child {
+	transform: none;
+	cursor: pointer;
+	color: var(--colour-primary);
+}
+
+.navbar-cta .badge-holder img {
+	flex-shrink: 0;
+}
+
+.profile-modal-holder {
+	position: relative;
+	flex-shrink: 0;
+}
+.profile-modal {
+	display: none;
+	position: absolute;
+	width: 15rem;
+	padding: 1rem;
+	right: 0;
+	background-color: var(--colour-accent-third);
+	margin-top: 1rem;
+	z-index: 2;
+	color: var(--colour-text-white-two);
+	border-radius: 1rem;
+	transition: all 0.3s ease-in-out;
+}
+
+.profile-modal::before {
+	content: '';
+	position: absolute;
+	top: -1.8rem;
+	right: 1rem;
+	margin-left: -5px;
+	border-width: 10px;
+	border-style: solid;
```

Comment:
maybe border-width can be converted into rem but you can keep it hard coded if thats the requirement.